<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationForm</class>
 <widget class="QWidget" name="NotificationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notifications</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="locationGroupBox">
     <property name="title">
      <string>Alerts</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="notifyBeforeCheckBox">
        <property name="text">
         <string>Display notifications before prayer times:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="notifyBeforeSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="notifyAfterCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Display notifications after prayer times:</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="notifyAfterSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="otherSettingsGroupBox">
     <property name="title">
      <string>Sound</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="enableSoundCheckBox">
        <property name="text">
         <string>Play athan at prayer times</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fileLocationLabel">
          <property name="text">
           <string>Adhan file:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KUrlRequester" name="soundSelect"/>
        </item>
        <item>
         <widget class="QToolButton" name="playStopToolButton">
          <property name="toolTip">
           <string>Play</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="fileLocationLabel_2">
          <property name="text">
           <string>Fajr Adhan file:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KUrlRequester" name="soundFajrSelect"/>
        </item>
        <item>
         <widget class="QToolButton" name="playStopFajrToolButton">
          <property name="toolTip">
           <string>Play</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="enableDuaaCheckBox">
          <property name="text">
           <string>Play Duaa</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>3</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KUrlRequester" name="soundDuaaSelect">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="playStopDuaaToolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Volume:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stopPlayersCheckBox">
     <property name="text">
      <string>Stop players at praying time (Amarok, Dragon Player...)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>notifyBeforeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifyBeforeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifyAfterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifyAfterSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileLocationLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playStopToolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileLocationLabel_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundFajrSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playStopFajrToolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>enableDuaaCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableDuaaCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundDuaaSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableDuaaCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playStopDuaaToolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
